src/pages/standard-report/HtmlReport.js

```diff
@@ -1,40 +1,68 @@
 import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+/**
+ * Do this stuff inside an iframe...
+ * https://stackoverflow.com/questions/10418644/creating-an-iframe-with-given-html-dynamically
+ */
 
 class HtmlReport extends Component {
+    state = {
+        ready: false,
+        report: '',
+    }
+
     // TODO: Fix urls/paths to scripts or find a better way to do this
     componentDidMount() {
```

Comment:
I know this is like polishing a 💩 but I have a few minor suggestions:
- Extract repeated logic for script loading to a single method i.e. `loadScript(src)`
- Then do something like this
```javascript
this.loadScript('/dhis-web-commons/javascripts/jQuery/jquery.min.js')
    .then(() => this.loadScript('/dhis-web-commons/javascripts/dhis2/dhis2.util.js'))
    .then(() => this.setState({ ready: true, report: this.props.html})
```
Quite likely you need to bind `this.loadScript` to `this` scope